src/blog/2022/11/flowforge-1-1-released.md

```diff
@@ -0,0 +1,62 @@
+---
+title: FlowForge 1.1 released
```

Comment:
Highlight the tag line feature in the title please

Reply:
Not sure what this means. I've reworked the whole blog post since this comment so I may have addressed your point in the latest version.

Reply:
For example;

> FlowForge 1.1 now includes persistent file storage

"FlowForge <version> released" is super dry IMO